core/arch/arm/kernel/secure_partition.c

```diff
@@ -1565,31 +1567,53 @@ static TEE_Result read_vm_availability_msg(const void *fdt,
 	return TEE_SUCCESS;
 }
 
+static TEE_Result get_boot_order(const void *fdt, uint32_t *boot_order)
+{
+	TEE_Result res = TEE_SUCCESS;
+
+	res = sp_dt_get_u32(fdt, 0, "boot-order", boot_order);
+
+	if (res == TEE_SUCCESS) {
+		if (*boot_order > UINT16_MAX) {
+			EMSG("Value of boot-order property (%" PRIu32
```

Comment:
Please keep the print string in one line, even over 80 columns.
Please remove the ` ` before and after `PRIu32` for consistency, and because it looks nice.

Reply:
Fixed.